docker/install-dependencies.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/bash
```

Comment:
Let's add `set -euo pipefail` here. See http://redsymbol.net/articles/unofficial-bash-strict-mode/.

This way, we can run `dnf clean all` in this script as well.

Reply:
Done.